sundials/CMakeLists.txt

```diff
@@ -296,7 +296,11 @@ if(BUILD_SHARED_LIBS AND WIN32)
     "#ifdef BUILD_SUNDIALS_LIBRARY
 #define SUNDIALS_EXPORT __declspec(dllexport)
 #else
+#ifdef LINK_SUNDIALS_STATIC
+#define SUNDIALS_EXPORT
+#else
```

Comment:
Just out of curiosity, what is this trying to do?

Reply:
Is it to be able to have both the shared and static versions of sundials libraries at the same time and use them selectively?

Reply:
It's from https://github.com/OpenModelica/OMCompiler-3rdParty/commit/8295908d1b04b72851ce7d33a1bead35a2962583.
I think with that on windows mingw we can build static and dynamic simultaniously.

Reply:
Ahh I see. Do we really need to have both? I guess there is a reason for it :)

Anyway if they are both needed you can try this maybe it solves your problem:

`#define SUNDIALS_EXPORT /*empty*/`
in all cases. 

Then use
`cmake -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE ....`
when building the shared libraries.

This way cmake will export all symbols even if they are not explicitly marked for dll exporting (`__declspec(dllexport)`).  It will make the import file a bit bigger and exports things that are not needed but it is the default on linux so I think we are okay with it.